scripts/admin/cmerge

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
Confusing: there are two "cmerge", one script and one build-in. Call this (c)merge-config-file.